packages/create-cloudflare/templates-experimental/angular/templates/wrangler.toml

```diff
@@ -3,3 +3,9 @@ name = "<TBD>"
 compatibility_date = "<TBD>"
 main = "./dist/server/server.mjs"
 assets = { directory = "./dist/browser", binding = "ASSETS" }
+
+# Workers Logs
+# Docs: https://rohin-observability-docs.cloudflare-docs-7ou.pages.dev/workers/observability/logs/workers-logs/
```

Comment:
These are all temporary links that I'll update when the docs PR gets merged before this PR is merged